lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
You can do that anyway, if you choose. Yay javascript. But `AppName.property` is a pretty common style for storing persistent objects in JS-heavy apps.